docs/Code-Base-Walkthrough.md

```diff
@@ -0,0 +1,200 @@
+Thanks for contributing to Brim! Here are a few pointers to help you understand the code base and feel confident making changes.
+
+
+
+## Entry Point
+
+A very high level overview of the boot process goes like this:
+
+The first file that gets run is `src/js/electron/main.js`. From there it will create a BrowserWindow and load the `search.html` file. That then loads JavaScript in  `src/js/search.js`. It creates the DOM needed to mount React, then renders the `src/js/components/App.js` component.
```

Comment:
```suggestion
The first file executed is `src/js/electron/main.js`. It will create a BrowserWindow and load the `search.html` file, which loads JavaScript in  `src/js/search.js`. `src/js/search.js` creates the DOM needed to mount React, then renders the `src/js/components/App.js` component.
```